modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -314,29 +314,65 @@ a|Print help information.
 a|`<device>`
```

Comment:
Out of scope but would you mind fixing line 310?
`s/coreos-install install subcommand/coreos-installer install subcommand`